Pods/Web3/Web3/Classes/Core/Transaction/EthereumAddress.swift

```diff
@@ -163,6 +166,19 @@ public struct EthereumAddress {
         return hex
     }
 
+    // https://github.com/ethereum/EIPs/blob/master/EIPS/eip-681.md
```

Comment:
this is a wrong place to modify the pod files. The pods will be re-generated by the next `pod install` or `pod update`, and these changes will be lost.

Also, adding this behavior to *all* decoding addresses from string, is plain wrong to me. Instead, it needs to be done in an extension to the Address in the Logic/Models/Address.swift